lib/evss/disability_compensation_form/form526_to_lighthouse_transform.rb

```diff
@@ -253,6 +255,21 @@ def transform_secondary_disabilities(disability_source)
           sd
         end
       end
+
+      def transform_direct_deposit(direct_deposit_source)
+        direct_deposit = Requests::DirectDeposit.new
+        if direct_deposit_source.nil?
```

Comment:
for this use case - missing a direct deposit node in the EVSS request structure - we agreed to **_not_** send this `'directDeposit'`node for now until the business logic is clear from stakeholders!
let's remove this node from the LH request body if there is no `directDeposit` node

Reply:
![PIm1apF - Imgur](https://github.com/department-of-veterans-affairs/vets-api/assets/92405130/e5ac8cf6-7429-488e-afa8-d3c6880dca26)

Reply:
@aurora-a-k-a-lightning this should be fixed now. my bad!